projects/cli-files/go-ls/cmd/root.go

```diff
@@ -0,0 +1,78 @@
+package cmd
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+)
+
+func Execute() {
+	// define a flag of type boolean with the specific properties (name, value, usage)
+	helpFlag := flag.Bool("h", false, "show help")
+
+	// parse the command line flags
+	flag.Parse()
+
+	// if the "-h" flag is provided
+	if *helpFlag {
+		fmt.Fprintf(os.Stdout, "go-ls help message\n")
+		return
+	}
+
+	// get the non-flag command line arguments
+	args := flag.Args()
+
+	// make a slice of paths to iterate over (if more than one argument is passed)
+	var paths []string
```

Comment:
This is a nice way to make the "no args" vs "some args" cases work the same in the rest of your code!